zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/SnapshotMetricsDoc.java

```diff
@@ -125,7 +127,47 @@ public Type getType() {
 
     @Override
     public KeyName[] getKeyNames() {
-      return PartitionKeyNames.values();
+      return getPartitionBootstrapKeyNames();
+    }
+  },
+  /** Approximate duration of snapshot transfer between two nodes */
+  SNAPSHOT_TRANSFER_DURATION {
+    @Override
+    public String getDescription() {
+      return "Total transfer duration of a snapshot between two nodes";
+    }
+
+    @Override
+    public String getName() {
+      return "zeebe.snapshot.transfer.duration";
+    }
+
+    @Override
+    public Type getType() {
+      return Type.TIMER;
+    }
+
+    @Override
+    public String getBaseUnit() {
+      return "ms";
+    }
+
+    @Override
+    public Duration[] getTimerSLOs() {
+      return new Duration[] {
+        Duration.ofMillis(50),
+        Duration.ofMillis(250),
+        Duration.ofMillis(500),
+        Duration.ofSeconds(1),
+        Duration.ofSeconds(5),
+        Duration.ofSeconds(15),
+        Duration.ofMinutes(30)
+      };
```

Comment:
:upside_down_face: Did you pick those by hand? I'd use more resolution in the seconds to minutes range and less in the millis. Or am I wrong in assuming that a transfer doesn't usually complete that fast?